common/install_clang.sh

```diff
@@ -9,11 +9,23 @@ if [ -n "$CLANG_VERSION" ]; then
     sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main"
   fi
 
-  sudo apt-get update
+  if [[  "$CLANG_VERSION" == 9 && "$UBUNTU_VERSION" == 18.04 ]]; then
+    wget --no-check-certificate -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add  -
+    apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-${CLANG_VERSION} main"
+  else
+      echo "Invalid Ubuntu version: ${UBUNTU_VERSION}"
+      exit 1
+  fi
 
+  sudo apt-get update
   apt-get install -y --no-install-recommends clang-"$CLANG_VERSION"
   apt-get install -y --no-install-recommends llvm-"$CLANG_VERSION"
 
+  # Install version of llvm and clang.
+  if [[ "$UBUNTU_VERSION" == 18.04 ]]; then
+    apt-get install -y --no-install-recommends llvm-"$CLANG_VERSION"-dev
+  fi
```

Comment:
This doesn't seem very intuitive: why is it that only Ubuntu 18.04 version is special cased to install llvm-dev? It would make more sense to introduce a different indicator, maybe "llvmdev", that controls whether or not these libraries get installed.